### PR TITLE
Anopheles diplotype clustering - fix n_snps in title

### LINKED ISSUE #556: Diplotype clustering title reports the number of amino acid snps, not total snps used in clustering
Important to report the total snps used in clustering in order to be confident of region / n snps

### PR DESCRIPTION
**Pull request by sanjaynagi**
* Resolves #556 

Currently its reporting the number of aa snps, not the number of SNPs used in clustering.

### CODE REVIEW

**Comment by sanjaynagi** [comment]
dohhhh forgot about the CI not working on forks again.

File: malariagen_data/anoph/dipclust.py [hunk 1]
```diff
@@ -631,7 +631,7 @@ def plot_diplotype_clustering_advanced(
                 subplot_heights.append(cnv_row_height * n_cnv_genes)
 
         if snp_transcript:
-            snp_trace, n_snps = self._dipclust_snp_trace(
+            snp_trace, n_snps_df = self._dipclust_snp_trace(
```
Comment (alimanfoo) [inline malariagen_data/anoph/dipclust.py:634]:
```suggestion
            snp_trace, n_snps_transcript = self._dipclust_snp_trace(
```

Nit, suggest renaming for even less ambiguity :)

File: malariagen_data/anoph/dipclust.py [hunk 2]
```diff
@@ -644,7 +644,7 @@ def plot_diplotype_clustering_advanced(
 
             if snp_trace:
                 figures.append(snp_trace)
-                subplot_heights.append(snp_row_height * n_snps)
+                subplot_heights.append(snp_row_height * n_snps_df)
```
Comment (alimanfoo) [inline malariagen_data/anoph/dipclust.py:647]:
```suggestion
                subplot_heights.append(snp_row_height * n_snps_transcript)
```

**Review by alimanfoo — COMMENTED** [review]
LGTM, thanks @sanjaynagi. Tiny suggestion...

**Review by alimanfoo — APPROVED** [review]
LGTM

**Comment by sanjaynagi** [comment]
@alimanfoo may need you to merge this I think! :)

**Pull request merged**